game-state-service

```diff
@@ -232,7 +238,18 @@ class GameStateService(Gio.Application):
         self._state.flush()
         Gio.Application.do_shutdown(self)
 
+    def do_activate(self):
+        pass
```

Comment:
It would be nice to have a comment here explaining why this method needs to be overriden with no logic.

Reply:
It was a left over from a version where I was calling `activate_action` which apparently needs this method defined. With the current version this is not needed so I removed it. Thanks for noticing it.